features/cli/policy.feature

```diff
@@ -165,38 +165,36 @@ Feature: change the policy of user/service account
 
   # @author chaoyang@redhat.com
   # @case_id OCP-10447
-  @admin
```

Comment:
I have removed tag `admin` from Polarion. Make sure to update `v3` branch as well because it will be a discrepancy in this way which can cause failures running smoke tests.

Reply:
@akostadinov PR for v3, https://github.com/openshift/verification-tests/pull/204